subdomains/blog/_posts/2020-12-21-announcing-volta-100.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "Announcing Volta 1.0.0"
+date:   2020-12-21 08:00:00 -0700
+categories: update
+author: Chuck Pierce
+excerpt_separator: <!--more-->
+---
+
+The Volta team is thrilled to announce that Volta 1.0.0 is now available! Volta is a fast, reliable utility for declaring your JavaScript DevTools as Code. In much the same way that Infrastructure as Code frees teams from manually managing their server infrastructure, **Volta takes the hassle out of defining and coordinating JavaScript tools** so that you can focus on creating awesome software.
```

Comment:
I think we're going with "Toolchains as Code" instead of "DevTools as Code" now (see the Discord discussion).

In the bold phrase I would replace "defining" with "installing".

I know we want to keep things short but I wonder if it would still be good to give examples like "**Volta takes the hassle etc etc JavaScript tools** such as Node, package managers, and package binaries, so that you can etc etc"